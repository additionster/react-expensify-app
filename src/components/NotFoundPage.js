import React from "react";
import {Link} from 'react-router';

const NotFoundPage = () => (
    <div>
        Page not Found - <Link to="/" className='is-active'>Go Home</Link>
    </div>
);
export default NotFoundPage;