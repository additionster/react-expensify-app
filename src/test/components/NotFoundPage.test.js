import React from "react";
import { render } from "@testing-library/react";
import '@testing-library/jest-dom';
import NotFoundPage from '../../components/NotFoundPage';
import { BrowserRouter } from "react-router";

test('shouldRenderNotFoundPageCorrectly', () => {
    const renderer = render(<BrowserRouter><NotFoundPage /></BrowserRouter>);
    expect(renderer.baseElement).toMatchSnapshot();
});