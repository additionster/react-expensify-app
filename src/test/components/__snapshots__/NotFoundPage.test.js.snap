// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`shouldRenderNotFoundPageCorrectly 1`] = `
<body>
  <div>
    <div>
      Page not Found - 
      <a
        class="is-active"
        data-discover="true"
        href="/"
      >
        Go Home
      </a>
    </div>
  </div>
</body>
`;
